cube/src/main/java/zone/cogni/asquare/cube/convertor/ModelToJsonConversion.java

```diff
@@ -717,19 +717,18 @@ private void addToJsonAsSingle(ObjectNode jsonNode, String attribute, JsonNode v
   }
 
   private String literalToDate(Literal literal) {
-    return date2string(ISODateTimeFormat.dateTimeParser().parseDateTime(literal.getLexicalForm()));
-  }
-
-  private String date2string(DateTime date) {
-    return date == null ? null : date.toString(ISODateTimeFormat.date());
+    return LocalDate.parse(literal.getLexicalForm()).toString();
   }
 
+  /**
+   * Tested on formats like 2016-01-01T00:00:00Z , 2021-02-11T08:19:21.489344Z and 2022-08-08T08:08:08.888+02:00
+   *
+   * @param literal dateTime literal
+   * @return dateTime string as is but after format validation
+   */
   private String literalToDateTime(Literal literal) {
```

Comment:
Not sure if it's important, but it seems we go from a dateTime with milliseconds to one with nanoseconds